Article/Article.js

```diff
@@ -3,20 +3,20 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
-  }
+    this.expandButton.addEventListener('click',() => this.expandArticle());
+  };
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.expandButton.classList.toggle('')
```

Comment:
You're almost there - just find the class that you need to toggle and pass it in here! Try looking in your Article.less file to find a class that might turn your articles on/off